cli/command/container/list.go

```diff
@@ -59,21 +59,12 @@ func newListCommand(dockerCli *command.DockerCli) *cobra.Command {
 	return &cmd
 }
 
-type preProcessor struct {
-	types.Container
```

Comment:
@AkihiroSuda The purpose of `map[string]bool` is to silently ignore the unknown fields for `preProcessor`. Even with an empty `map[string]bool`, no error will be raised in `Execute()` for `CreatedAt`, `Networks`, etc.

At the same time, we want to find out if field `.Size` has been specified  in the pre-processing step. So an `func Size()` has been added to set the value of the map and get this value after pre-processing is done:
```
func (p preProcessor) Size() bool {
	p["size"] = true
	return true
}
```